lib/generators/shopify_app/scopes_storage/scopes_storage_generator.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+require 'rails/generators/base'
+require 'rails/generators/active_record'
+
+module ShopifyApp
+  module Generators
+    class ScopesStorageGenerator < Rails::Generators::Base
+      include Rails::Generators::Migration
+      source_root File.expand_path('../templates', __FILE__)
+
+      class_option :with_cookie_authentication, type: :boolean, default: false
+      class_option :embedded, type: :string, default: 'true'
+
+      def create_scopes_storage_in_shop_model
+        if yes?("Do you want to add scopes column to the Shop model? [y/n]")
```

Comment:
I wonder if this prompt may seem unclear to a developer without context, but I can't think of a suggestion other than mentioning it's recommended for embedded apps.

Reply:
Hmm, maybe this string could provide a little more context as to why this might be needed for some developers.

```suggestion
        scopes_column_prompt = <<~PROMPT
          It is highly recommended that embedded apps store the access scopes granted by merchants during app installation. The following migration will add a `scopes` column to the Shop model.
          
          Do you want to add this migration? [y/n]
        PROMPT
        
        if yes?(scopes_column_prompt)
```

Reply:
I think that part of this PR should help outline a clear path forward that makes this the default output of our generators in future versions.